openresty/Dockerfile-alpine

```diff
@@ -0,0 +1,254 @@
+# Current latest Openresty version is here.
+# Also specify the nginx core used. (Openresty, adds their own version numbers after the nginx one, so we can't use the same number for both Openresty and nginx)
+# Also specify the LUA version used. (Comes with the latest Openresty src bundle.)
+ARG OPENRESTY_VERSION="1.21.4.3" \
+    NGINX_VERSION="1.21.4" \
+    LUA_JIT_VERSION="2.1-20230410" \
+    MODSEC_VERSION="3.0.11"
+
+FROM openresty/openresty:${OPENRESTY_VERSION}-alpine-fat as build
+
+ARG MODSEC_VERSION \
+    OPENRESTY_VERSION \
+    NGINX_VERSION \
+    LUA_JIT_VERSION 
+
+# Remove pcre2 compliation of ModSecurity for openresty, until openresty gets updated to support pcre2. Relevant github issue: https://github.com/openresty/lua-nginx-module/issues/1984
```

Comment:
```suggestion
# Remove pcre2 compilation of ModSecurity for openresty, until openresty gets updated to support pcre2. Relevant github issue: https://github.com/openresty/lua-nginx-module/issues/1984
```

Reply:
Because nginx was updated to 1.25 for openresty I will revisit the issue. The reason for openresty using 1.21 of nginx was because that was the last version to support pcre. The change to nginx 1.25 means that they have overcome that issue and I will need to change the compilation to use pcre2.